internetbs.php

```diff
@@ -2390,8 +2390,8 @@ private function getRawTldData($module_row_id = null)
 
             // Get domain price list
             $price_list = $command->getPriceList(['version' => 2]);
+            $this->processResponse($api, $price_list);
             $response = $price_list->response();
-            $this->processResponse($api, $response);
 
             // Save pricing in cache
             if (Configure::get('Caching.on') && is_writable(CACHEDIR)) {
```

Comment:
We should also probably make this caching conditional on a successful return (i.e. $price_list->status() == 200)